Manatee.Trello/Contracts/IBoardBackgroundCollection.cs

```diff
@@ -8,12 +8,12 @@ namespace Manatee.Trello
 	/// </summary>
 	public interface IBoardBackgroundCollection : IReadOnlyCollection<IBoardBackground>
 	{
-		/// <summary>
-		/// Adds a custom board background.
-		/// </summary>
-		/// <param name="data">The byte data of the file to attach.</param>
-		/// <param name="ct">(Optional) A cancellation token for async processing.</param>
-		/// <returns>The newly created <see cref="IBoardBackground"/>.</returns>
-		Task<IBoardBackground> Add(byte[] data, CancellationToken ct = default);
+        /// <summary>
+        /// Adds a custom board background.
+        /// </summary>
+        /// <param name="filePath">The path of the file to attach.</param>
+        /// <param name="ct">(Optional) A cancellation token for async processing.</param>
+        /// <returns>The newly created <see cref="IBoardBackground"/>.</returns>
+        Task<IBoardBackground> Add(string filePath, CancellationToken ct = default);
```

Comment:
Instead of changing this, can you just add the new method, please?  This will allow us to avoid a breaking change.

Reply:
Alright, its seems to be a great idea